tests/inputs/mvs_config.json

```diff
@@ -637,6 +637,14 @@
                     "unit": "factor",
                     "value": 0.2
                 },
+                "fixed_thermal_losses_relative": {
```

Comment:
The changelog should also mention that you changed this file! 

It connects to #750, as I would usually not want these two parameters to be in here, but know they are necessary at this point.

Reply:
Done with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/2090e8d17a1cd526d31e9f2fd545799904154f22

Reply:
Ah, I see, thats why you want it here. Still, adding it here but not in the csv I can ensure you that this info will be lost, and no-one knows why there are new errors. So either it also has to be in the csvs, or we need to make an exception for those two parameters in the pytests. @Bachibouzouk would you know where to put the exception?